deepmd/common.py

```diff
@@ -125,6 +125,7 @@ def gelu_wrapper(x):
     "softplus": tf.nn.softplus,
     "sigmoid": tf.sigmoid,
     "tanh": tf.nn.tanh,
+    "swish": tf.nn.swish,
```

Comment:
It seems that it has been renamed to `silu`: https://github.com/tensorflow/tensorflow/issues/41066